src/components/create-courses/component/authors-lists.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {useMemo} from "react";
+import { Button, List, Col, Row } from "reactstrap";
+
+export default function AuthorsLists({ authorsList, setAuthorsList }) {
+
+	const authorsNotSelected = useMemo( () => {
+		return authorsList.filter(({ selected }) => !selected);
+	}, [authorsList])
+
+	const authorsSelected = useMemo( () => {
+		return authorsList.filter(({ selected }) => selected);
+	}, [authorsList])
+
+	function selectToggle(authorId) {
+		const newArray = authorsList.filter((author) => author.id !== authorId);
+		const author = authorsList.find(author => author.id === authorId);
+		author.selected === true ?	author.selected = false : author.selected = true;
+		newArray.push(author)
+		setAuthorsList(newArray)
+	}
+
+	return <Row xs='1' className='mt-1'>
+		{!authorsNotSelected.length ? <List type="unstyled">
+				<li><b>Available authors empty</b></li>
+			</List> : <List type="unstyled">
+				<b>Available authors:</b>
+				{authorsNotSelected.map((author) => <li  className='row mt-1' key={author.id}>
```

Comment:
You can operate with authorList from props and use ternary operator to check the render `selected ? <... /> : null`

Reply:
Done